### PR TITLE
Add second screenshot to performance email

### PR DESCRIPTION
**Pull request by aleksbykov**
- cluster.py BaseMonitorSet.get_grafana_screenshot:
  - Adding list of urls.
  - Add loop over the list of urls to create screen
shots.
  - Return value now is list of links to screenshots
- results_analyze.py PerformanceResultAnalyzer._get_grafana_snapshots
  - Return value is now list.
  - For backcompatibility, added checks if value is string, return list
- results_performance.html
  - Added the For loop to process the list of grafana links

### CODE REVIEW

File: sdcm/results_performance.html [hunk 1]
```diff
@@ -98,11 +98,15 @@ <h3>Links:</h3>
         <li><a href={{ dashboard_master }}>Kibana</a></li>
         <li><a href="{{ job_url }}">Jenkins job</a></li>
         {% if grafana_snapshot %}
-        <li><a href={{ grafana_snapshot }}>Download Grafana Screenshot</a></li>
+        {% for snapshot in grafana_snapshot %}
+        <li><a href={{ snapshot }}>Download Grafana Screenshot</a></li>
```
Comment (bentsi) [inline sdcm/results_performance.html]: We need to know which screenshot we are downloading. Please change it to include dashboard name.
To implement this you can, for instance, add the dashboard name to the screenshot filename and then deduce it here or when rendering template
The link, for instance, can have the following name: Download "Per server metrics nemesis" dashboard screenshot*

Comment (aleksbykov) [reply sdcm/results_performance.html]: Add the name of dashboard to filename and change the template.

**Comment by bentsi** [comment]
@roydahan should we backport this to `branch-perf-v3` and `branch-3.0`?

**Comment by roydahan** [comment]
Branch-perf-v5 only for now.

**Comment by bentsi** [comment]
backported to `Branch-perf-v5`

**Pull request merged**